src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -105,6 +129,168 @@ class UserService(protected val userInfo: UserInfo, dataSource: DataSource, prot
     }
   }
 
+  def listAdmins() = {
+    asAdmin {
+      gcsDAO.listAdmins.map(users => RequestComplete(StatusCodes.OK, UserList(users))).recover{ case throwable =>
+        RequestComplete(ErrorReport(StatusCodes.BadGateway,"Unable to list admins.",gcsDAO.toErrorReport(throwable)))
+      }
+    }
+  }
+
+  def isAdmin(userId: String) = {
+    asAdmin {
+      tryIsAdmin(userId) map { admin =>
+        if (admin) RequestComplete(StatusCodes.NoContent)
+        else RequestComplete(ErrorReport(StatusCodes.NotFound, s"User ${userId} is not an admin."))
+      } recover { case throwable =>
+        RequestComplete(ErrorReport(StatusCodes.BadGateway, s"Unable to determine whether ${userId} is an admin.", gcsDAO.toErrorReport(throwable)))
+      }
+    }
+  }
+
+  def addAdmin(userId: String) = {
+    asAdmin {
+      tryIsAdmin(userId) flatMap { admin =>
+        if (admin) {
+          Future.successful(RequestComplete(StatusCodes.NoContent))
+        } else {
+          gcsDAO.addAdmin(userId) map (_ => RequestComplete(StatusCodes.Created)) recover { case throwable =>
+            RequestComplete(ErrorReport(StatusCodes.BadGateway, s"Unable to add ${userId} as an admin.", gcsDAO.toErrorReport(throwable)))
+          }
+        }
+      } recover { case throwable =>
+        RequestComplete(ErrorReport(StatusCodes.BadGateway, s"Unable to determine whether ${userId} is an admin.", gcsDAO.toErrorReport(throwable)))
+      }
+    }
+  }
+
+  def deleteAdmin(userId: String) = {
+    asAdmin {
+      tryIsAdmin(userId) flatMap { admin =>
+        if (admin) {
+          gcsDAO.deleteAdmin(userId) map (_ => RequestComplete(StatusCodes.NoContent)) recover { case throwable =>
+            RequestComplete(ErrorReport(StatusCodes.BadGateway, s"Unable to delete ${userId} as an admin.", gcsDAO.toErrorReport(throwable)))
+          }
+        } else {
+          Future.successful(RequestComplete(ErrorReport(StatusCodes.NotFound,s"${userId} is not an admin.")))
+        }
+      } recover { case throwable =>
+        RequestComplete(ErrorReport(StatusCodes.BadGateway, s"Unable to determine whether ${userId} is an admin.", gcsDAO.toErrorReport(throwable)))
+      }
+    }
+  }
+
+  def listGroupMembers(groupName: String) = {
```

Comment:
I have it group name so the endpoint isn't something like:

rawls.dsde-dev.broadinstitute.org/api/admin/groups/group_era-commons@firecloud.org/members
 and is instead something more beautifuler like /api/admin/groups/era-commons/members